packages/flutter/lib/src/material/input.dart

```diff
@@ -10,12 +10,20 @@ import 'debug.dart';
 import 'icon.dart';
 import 'icon_theme.dart';
 import 'icon_theme_data.dart';
-import 'material.dart';
 import 'text_selection.dart';
 import 'theme.dart';
 
 export 'package:flutter/services.dart' show TextInputType;
 
+const Duration _kTransitionDuration = const Duration(milliseconds: 200);
+const Curve _kTransitionCurve = Curves.fastOutSlowIn;
+
+/// A simple text field.
+///
+/// See also:
+///
+/// * [Input]
```

Comment:
add a brief explanation of why you're pointing to these, as in ` * [Input], which does this and that.`